client/src/App.js

```diff
@@ -1,20 +1,16 @@
 import { Route, Switch } from "react-router-dom";
 
-import About from "./pages/About";
+// import About from "./pages/About";
 import Home from "./pages/Home";
-import BookingScreen from "./pages/BookingScreen";
 
 const App = () => (
 	<Switch>
 		<Route path="/" exact>
 			<Home />
 		</Route>
-		<Route path="/booking">
-			<BookingScreen bookingDate="2022-01-18T00:00:00.000Z"/>
-		</Route>
-		<Route path="/about/this/site">
+		{/* <Route path="/about/this/site">
```

Comment:
Why commenting this out?

Reply:
I didn't, It's green because I remove some lines before and it's comes upper. Just in case someone else using this for other feature I didn't remove this.